docs/contributing.md

```diff
@@ -33,21 +33,94 @@ The next Community Call will be held on October 24th, 2024 at [5:00 PM CEST](htt
 Please use the [project's issue tracker](https://github.com/headlamp-k8s/headlamp/issues) for filing any bugs you find or features
 you think are useful.
 
+### Guidelines for Submitting an Issue
+
+When submitting an issue, follow these guidelines to help maintainers address it efficiently:
+
+**Search for Existing Issues:**
+
+- Use the issue tracker to see if your issue already exists. If it does, consider adding your input to the existing issue instead of opening a new one.
+- If the issue doesn't exist, please create a new issue using one of the templates available (bug, feature, security, or blank).
+
 ### Security issues
 
 For filing security issues that are sensitive and should not be public, please
 send an email to <security@headlamp.dev> .
 
-## Translations
+## Submitting a Pull Request (PR)
 
-If you want to contribute to the internationalization of Headlamp, please refer to the
-dedicated [i18n docs](./development/).
+Follow these steps when submitting a PR to ensure it meets the project’s standards.
 
-### Complex contributions
+### 1. Run Tests and Format Your Code
 
-If you have a complex contribution in mind (meaning changes in the architecture
-or a lot of LOC changed), it is advisable to first file a GitHub issue and
-discuss the implementation with the project's maintainers.
+Navigate to the `frontend` folder and run the following commands:
+
+- `cd frontend`
+- `npm run test` – Run the test suite.
+- `npm run tsc` – Check for TypeScript errors.
+- `npm run format` – Format your code to match the project’s style.
```

Comment:
I thought we had `make frontend-lint` and `make frontend-test` for making it simpler. Maybe we suggest these instead?

Reply:
done!